app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -71,8 +69,8 @@ class MainActivity : AppCompatActivity() {
                     val jsonObjectMain = jsonResponse.getJSONObject("main")
                     //Get the values of temp, pressure, and humidity from Json object main
                     //We subtract it to get temperature in celsius
-                    val temp = jsonObjectMain.getDouble("temp") - 273.15
-                    val feelsLike = jsonObjectMain.getDouble("feels_like") - 273.15
+                    val temp = jsonObjectMain.getDouble("temp") - TEMP_NUMBER
+                    val feelsLike = jsonObjectMain.getDouble("feels_like") - TEMP_NUMBER
```

Comment:
Here is a link for more information on constants.
https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods